examples/bench/.gitignore

```diff
@@ -0,0 +1 @@
+/bench
```

Comment:
What is this?

Reply:
if I run "go build", I don't want git saying "hey hey there's a file called bench here, do you want me to add it?"

Reply:
OK you might need this but let's not add .gitignore for consistency.